doc/source/reference/routines.indexing.rst

```diff
@@ -43,6 +43,7 @@ Indexing-like operations
    diagonal
    select
    lib.stride_tricks.as_strided
+   lib.stride_tricks.sliding_window_view
```

Comment:
```suggestion
   sliding_window_view
```
Note that `as_strided` is deliberately _not_ available as `np.as_strided` as it is not present in `__all__`.

I think it's fine that you have `sliding_window_view` in `__all__` in this PR, but you therefore shouldn't have the namespacing here.

Reply:
Agreed. I added this before we put the array dispatch in place and forgot to update.

I am then also moving it in front of `as_strided` in this list, correct?